src/vs/workbench/contrib/terminal/browser/media/GitTabExpansion.psm1

```diff
@@ -0,0 +1,772 @@
+# ---------------------------------------------------------------------------------------------
```

Comment:
Does this file get signed so that users whose ExecutionPolicy is set to the default value (usually AllSigned) will work?

Reply:
Same as above, if you have an idea on how to get this work it would be helpful